lua/GPM.lua

```diff
@@ -325,7 +325,7 @@ end
 GPM_ = {
 	type_ = "GPM",
 	--- Create attributes in the origin according to the relations established by GPM.
-	-- These attributes are created in memory, and must be saved manually if needed.
+	-- These attributes are created in memory, and must be saved manually afterwards if needed.
```

Comment:
1. "all": what would be the attribute created value?
1.  "average": which would be this weights?
1. "maximum and minimum":  I don't understand these documentation. Wouldn't they associated with distance?
1. "max": what would be greater than math.huge?
1. in general: I think that GPM documentation is not enough to understand its concepts, I suggest a documentation on Wiki with the details of it and its definitions.